apps/src/netsim/NetSimDnsTable.html.ejs

```diff
@@ -0,0 +1,44 @@
+<%
+var NetSimRouterNode = require('./NetSimRouterNode');
+var DnsMode = NetSimRouterNode.DnsMode;
+%>
+<div class="netsim_dns_table">
+  <h1>My Network</h1>
+  <table>
+    <thead>
+    <tr>
+      <th>Hostname</th>
+      <th>Address</th>
+    </tr>
+    </thead>
+    <tbody>
+    <%
+    tableData.forEach(function (row) {
+      var displayHostname = row.hostname;
+      var displayAddress = '';
+      var rowClasses = [];
+
+      if (dnsMode === DnsMode.NONE || row.isDnsNode || row.isLocal) {
+        displayAddress = row.address;
+      }
+
+      if (row.isLocal) {
+        displayHostname += " (Me)";
```

Comment:
I'm trying to keep it in mind, but I've been told not to worry about loc during this first iteration, and that we are not likely to RTL netsim anytime soon.  I'm not very familiar with our loc pipeline.  Is there a specific change here you'd recommend to be prepared for that?